package.json

```diff
@@ -32,20 +39,28 @@
   },
   "homepage": "https://github.com/nickcis/polygonize#readme",
   "devDependencies": {
+    "@std/esm": "^0.11.3",
     "babel-cli": "^6.24.1",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
     "benchmark": "^2.1.4",
     "eslint": "^3.19.0",
     "eslint-config-mourner": "^2.0.1",
     "load-json-file": "^2.0.0",
+    "rimraf": "^2.6.2",
     "tape": "^4.6.3",
     "write-json-file": "^2.2.0"
   },
   "dependencies": {
-    "@turf/envelope": "^4.3.0",
-    "@turf/helpers": "^4.3.0",
-    "@turf/inside": "^4.3.0",
-    "@turf/invariant": "^4.3.0",
-    "@turf/meta": "^4.3.0"
+    "@turf/envelope": "*",
+    "@turf/helpers": "*",
+    "@turf/inside": "*",
+    "@turf/invariant": "*",
+    "@turf/meta": "*"
```

Comment:
👍 Looks good, none of these modules will have any "breaking" change to your repo, so we are going to be fine once they all get bumped up to v5.0